### ISSUE TITLE
How do you switch Propshaft back to dynamic mode in Development?

### ISSUE THREAD

**Issue by ethan-dowler** [rails/propshaft#97]
The docs say:
> **Precompilation in development**
> 
> Propshaft uses a dynamic assets resolver in development mode. However, when you run assets:precompile locally Propshaft will then switch to a static assets resolver. Therefore, changes to assets **will not be observed anymore** and you will have to precompile the assets each time changes are made. This is different to Sprockets.

My question is:
How do I switch it back? It seems crazy that if you run `rails assets:precompile` one time you're screwed... 🙁

**Comment by brenogazzola** [comment]
Just delete the `public/assets` folder. Or modify your `bin/dev` file do add this line before the `foreman` command:
```
rm public/assets/.manifest.json
```

**Comment by abevoelker** [comment]
I ran into this today as well - I couldn't figure out why my browser was seeing stale assets. Traced it back to my `public/assets` directory, then ultimately realized I must've ran `assets:precompile` just while playing around with Propshaft. Running `rails assets:clobber` is another way to "fix" the problem as the whole `public/assets` dir gets deleted.

Perhaps when precompiling assets when `Rails.env.development?` a warning message should be displayed that dynamic changes to assets will no longer be rendered until the manifest file is deleted?

**Comment by brenogazzola** [comment]
@abevoelker Yup, adding a warning there is on the list of things to be done. I wouldn't reject a PR for it 😉
